dvc/remote/gdrive.py

```diff
@@ -46,7 +46,10 @@ def should_retry(exc):
             return False
 
         retry_codes = [403, 500, 502, 503, 504]
-        return exc.error.get("code", 0) in retry_codes
+        result = exc.error.get("code", 0) in retry_codes
+        if result:
+            logger.debug("Retry GDrive API call failed with {}.".format(exc))
```

Comment:
can we put a counter here (Retrying 2 time, API call failed with ... format(exc)`?

Reply:
retry attempt seems is internal variable of funcy.retry decorator.
we can use some global counter variable, but seems it is hard to reset such counter in case if retry attempt was successful; Also not sure if there Python will make a copy of global variable per worker thread, or all threads will reference the same counter and increase incorrectly.

Reply:
yes, not need for the global one ... let's keep it this way for now ... @Suor - it's a feature request for the future (or might be you know some better way to get this information - more context on what we are retrying and how many times)